### PR TITLE
Mk add outcome

### PR DESCRIPTION
**Pull request by mkangia**
cc: @proteusvacuum 

1. Add new treatment outcomes
2. Send '0' as phone number where not available

- [x] Re-trigger failed cases

### CODE REVIEW

**Review by emord — APPROVED** [review]

File: custom/enikshay/integrations/nikshay/repeater_generator.py [hunk 1]
```diff
@@ -568,7 +568,8 @@ def _get_person_case_properties(episode_case, person_case, person_case_propertie
         # 2B is currently setting age_entered but we are in the short term moving it to use age instead
         "page": _get_person_age(person_case_properties),
         "paddress": person_case_properties.get('current_address', ''),
-        "pmob": person_case_properties.get(PRIMARY_PHONE_NUMBER, ''),
+        # send 0 since that is accepted by Nikshay for this mandatory field
+        "pmob": person_case_properties.get(PRIMARY_PHONE_NUMBER, '0'),
```
Comment (emord) [inline custom/enikshay/integrations/nikshay/repeater_generator.py]: Is this related to the treatment_outcome? I don't see a 0 there?

**Review by emord — APPROVED** [review]
comment is non blocking assuming you don't need the 0 there

File: custom/enikshay/integrations/nikshay/repeater_generator.py (hunk 1 above)
Comment (mkangia) [reply custom/enikshay/integrations/nikshay/repeater_generator.py]: Sneaked in an unrelated small fix that came in yesterday for missing phone number.

`send 0 since that is accepted by Nikshay for this mandatory field`

**Review by sravfeyn — APPROVED** [review]

File: custom/enikshay/integrations/nikshay/repeater_generator.py (hunk 1 above)
Comment (proteusvacuum) [reply custom/enikshay/integrations/nikshay/repeater_generator.py]: should we be trying to send any of the alternate phone numbers here if this specific property isn't set? We collect all of these:
```
PRIMARY_PHONE_NUMBER = 'phone_number'
BACKUP_PHONE_NUMBER = 'secondary_contact_phone_number'
OTHER_NUMBER = "phone_number_other"
ALTERNATE_NUMBER_1 = "phone_number_alternate_1"
ALTERNATE_NUMBER_2 = "phone_number_alternate_2"
ALTERNATE_NUMBER_3 = "phone_number_alternate_3"
```

Comment (mkangia) [reply custom/enikshay/integrations/nikshay/repeater_generator.py]: We already send `secondary_contact_phone_number` in the payload.
The reason for sending `0` was to notify that we actually don't have the phone number i.e notifying the actual scenario. Probably would have fallen back to other numbers if it was mandatory.

Comment (proteusvacuum) [reply custom/enikshay/integrations/nikshay/repeater_generator.py]: Yeah. I mean if `phone_number` isn't set, maybe one of the alternate numbers _is_ and we should be sending that one as the "primary" number instead?

Comment (mkangia) [reply custom/enikshay/integrations/nikshay/repeater_generator.py]: hmm, 
i checked two cases which had phone number missing, these were missing all alternate numbers as well. Looks like its highly unlikely someone would add an alternate number but not add the primary phone number unless the apps allows something funky.
Happy to add the checks if you think it would be of value.

Comment (proteusvacuum) [reply custom/enikshay/integrations/nikshay/repeater_generator.py]: That makes sense. I guess fine to leave as is if you think it doesn't add anything.
